### ISSUE TITLE
[Social - Sponsor] Create post for Gadfly

### ISSUE THREAD

**Issue by Readpato** [Schrodinger-Hat/osday-2023#548]
Text below from here: https://discord.com/channels/963080048380895352/1309505897508241439

Perfect, the sponsorship is sent and should be available Thursday at the latest, and you can review the transfer status via this link. Can't believe it's only 100 days away now! 

Here's a description for Gadfly, and our Logo is attached - you can go with plain or fancy, whichever suits your vibes.

Gadfly AI is a collective of open-source developers and artists dedicated to making the digital world more accessible and inclusive. Through our advocacy, training, and creative initiatives, we bridge the gap between technology and community. Online, we offer courses that blend accessibility and disability advocacy with open-source design, development, and scientific expertise. Offline, our printed zines bring Open Source into the hands of readers at local, independent bookstores. Our projects explore the frontiers of technology, from space exploration to computational literacy, with a strong commitment to digital accessibility. At Gadfly AI, we’re passionate about fostering a future where Open Source serves everyone—empowering individuals, inspiring innovation, and transforming the way we learn and create together.

link: https://gadfly.ai/

I loghi si trovano nella cartella drive di OSDay25 -> Sponsors -> Gadfly

**Comment by Readpato** [comment]
Two options (I like more the first one)

✨ Gadfly AI x Open Source Day 2025: A Gold Partnership for Change!
We're thrilled to announce our first Gold Sponsor for OSDay 2025 – Gadfly AI, a remarkable collective redefining open source through action. Their unique approach blends digital innovation with community empowerment, creating a bridge between technology and society. Through their accessibility-focused online courses and creative initiatives, they're transforming how we learn and interact with technology. Their innovative printed zines, found in local bookstores, bring open source knowledge directly to communities, while their digital projects span from space exploration to computational literacy.
🎯 Innovation in action:

Groundbreaking space exploration projects
Advanced computational literacy initiatives
Cutting-edge accessibility solutions

🌍 Community impact:

Local bookstore partnerships
Accessibility-focused training programs
Artist-developer collaborations

Join us in Florence on March 21, 2025! Together with Gadfly AI, we're building a future where technology truly serves everyone. Be part of this transformative journey!
Learn more about Gadfly AI: https://gadfly.ai/
Details & Registration: [Event Link]
#OSDay2025 #OpenSource #DigitalInclusion #TechCommunity

<hr> 

✨ Gadfly AI x Open Source Day 2025: A Gold Partnership for Change!
We're thrilled to announce our first Gold Sponsor for OSDay 2025 – Gadfly AI, a remarkable collective redefining open source through action. Their unique approach blends digital innovation with community empowerment, from offering accessibility-focused online courses to publishing tech zines in local bookstores.
🎯 Innovation in action:

Groundbreaking space exploration projects
Advanced computational literacy initiatives
Cutting-edge accessibility solutions

🌍 Community impact:

Local bookstore partnerships
Accessibility-focused training programs
Artist-developer collaborations

Join us in Florence on March 21, 2025! Together with Gadfly AI, we're building a future where technology truly serves everyone. Be part of this transformative journey!
Learn more about Gadfly AI: https://gadfly.ai/
Details & Registration: [Event Link]
#OSDay2025 #OpenSource #DigitalInclusion #TechCommunity

<hr>

Images ![Image](https://github.com/user-attachments/assets/f9cb42a6-946e-445a-9a37-c9c139e65669)
 proposal:


or 

![Image](https://github.com/user-attachments/assets/690145df-57ef-4fb9-87bf-e5802343e68b)

**Comment by TheJoin95** [comment]
First one is good, also the first image is good as well.

**Comment by TheJoin95** [comment]
note: we should identify a social post design for osday to announce the partnership, speakers, agenda and we should stick with it cc @LobeTia @AndreaChiara-commits @404answernotfound

**Comment by Readpato** [comment]
I agree. Do we wanna wait with this post then?

**Comment by AndreaChiara-commits** [comment]
I don't know what Gadfly is. For OSDay, I sent some sample yesterday, but only got a feedback from Patrick. Is on the social media thread on discord.

**Comment by TheJoin95** [comment]
> I agree. Do we wanna wait with this post then?

I'd say we should discuss this on monday with the other

> I don't know what Gadfly is. For OSDay, I sent some sample yesterday, but only got a feedback from Patrick. Is on the social media thread on discord.

I tend to agree with pato's feedback on that

Let's see what the others think on the subject, so let's not schedule the post until we have a clear vision on the topic

**Comment by AndreaChiara-commits** [comment]
Sure. On Monday when?

**Comment by cristianocasella** [comment]
Can we aggregate a little better the topics? I feel a little bit lost between the tools.

The first point is the design strategy, for this we should collect all the proposal in a single place and agree for a final one. I remember also a thread where the final pattern was just committed to the artist, so we have also an external dependency here.

Once the first topic has been sorted, we need to collect all the sponsors that need to be announced and create a sort of schedule, to avoid announcing them unsorted or at not profitably time.

Any thoughts? Happy to organize the two threads

**Comment by TheJoin95** [comment]
Let's tackle the discussion via Discord by opening a post in the forum; in the recent grooming session I feel we did not have enough light on this topic so I agree with you Cristiano. Could you open it?

**Comment by Readpato** [comment]
I agree with the point of the design proposals, that's why we are having the meeting today at 18 of social media to decide those.

I would also keep using discord until we find a better tool to organize ourselves.

Regarding the schedule you can look here https://github.com/Schroedinger-Hat/schroedinger-hat/issues/286#issuecomment-2494218109 for the established schedule as of now of social posts. I would utilize that one to keep track of the posts.

**Comment by Readpato** [comment]
This should be tackled with the new design proposed by @AndreaChiara-commits and discussed with me and @LobeTia, we should do this in the upcoming week/s

**Comment by TheJoin95** [comment]
Gadfly might send us a better logo in SVG in case you need it. The design should be the same.

The SVG logo could be good to add it in the sponsors table on the websites

**Comment by Readpato** [comment]
Awaiting OSDay graphics design from @LobeTia, most probably will arrive next week, we can already start preparing the text for the posts

**Comment by Wabri** [comment]
Non lo trovo su plane, ha un nome diverso il task? Stavo facendo pulizia e vorrei evitare di duplicare o altro.

**Comment by TheJoin95** [comment]
https://app.plane.so/schroedinger-hat/projects/8fcb2221-1321-484a-b500-b6ea118daab2/issues/816607a8-078d-4ad2-98ec-17d6f865a528
